libraries/chain/combined_database.cpp

```diff
@@ -204,7 +204,7 @@ namespace eosio { namespace chain {
             // that will concurrently perform a compaction job by
             // breaking it into multiple,
             // smaller ones that are run simultaneously.
-            options.max_subcompactions = cfg.rocksdb_threads;	// Default should be the # of CPUs
+            options.max_subcompactions = cfg.persistent_storage_num_threads;	// Default should be the # of CPUs
```

Comment:
@victorj8 - we should remove the comment on this line as it applied to the earlier setting and is confusing.

Reply:
Done